frame/support/src/lib.rs

```diff
@@ -1638,10 +1668,16 @@ pub mod pallet_prelude {
 /// pub enum Error<T> {
 /// 	/// $some_optional_doc
 /// 	$SomeFieldLessVariant,
+/// 	/// $some_more_optional_doc
+/// 	$SomeSingleFieldVariant(FieldType),
 /// 	...
 /// }
 /// ```
-/// I.e. a regular rust enum named `Error`, with generic `T` and fieldless variants.
+/// I.e. a regular rust enum named `Error`, with generic `T` and fieldless or single-field
+/// variants.
+/// Any field in the enum variants must implement `scale_info::TypeInfo` in order to be
```

Comment:
Yes, but we actually don't want any manual implementations of `PalletError`, because that would easily break the 4-byte limit if they're not careful with it. I need to instead say that they need to derive it.